doc/lua_api.txt

```diff
@@ -2211,7 +2212,80 @@ Elements
 **Note**: do _not_ use a element name starting with `key_`; those names are
 reserved to pass key press events to formspec!
 
+### `style[<name>;<prop1>;<prop2>;...]`
 
+Set the style for the named element `name`.
+Note: this **must** be before the element's tag.
+
+See [Styling Formspecs].
+
+
+### `style_type[<type>;<prop1>;<prop2>;...]`
+
+Sets the style for all elements of type `type` which appear after this tag.
+
+See [Styling Formspecs].
+
+
+
+Styling Formspecs
+-----------------
+
+Formspec elements can be themed using the style tags:
+
+	style[<name>;<prop1>;<prop2>;...]
+	style_type[<type>;<prop1>;<prop2>;...]
+
+Where a prop is:
+
+	property_name=property_value
+
+For example:
+
+	style_type[button;bgcolor=#006699]
+	style[world_delete;bgcolor=red;textcolor=yellow]
+	button[4,3.95;2.6,1;world_delete;Delete]
+
+Setting a property to nothing will reset it to the default value. For example:
+
+	style_type[button;bgimg=button.png;bgimg_pressed=button_pressed.png;border=false]
+	style[btn_exit;bgimg=;bgimg_pressed=;border=;bgcolor=red]
+
+
+### Valid Properties
+
+* button, button_exit
+	* bgcolor - color, sets button tint
+	* textcolor - color, default white
+	* border - boolean, draw border. Set to false to hide the bevelled button pane.  Default true.
+	* noclip - boolean, set to true to show things over the element bounds. Default false.
+	* bgimg - standard image. Defaults to none.
+	* bgimg_pressed - image when pressed. Defaults to bgimg when not provided.
+	* alpha - boolean, whether to draw alpha in bgimg. Default true.
```

Comment:
Please indent these properties the same way as in `ColorSpec` (or various other places)